src/Core/tests/DeviceTests/Services/FontManagerTests.Android.cs

```diff
@@ -21,11 +22,21 @@ public partial class FontManagerTests : TestBase
 	public void CanLoadFonts(string fontName, string assetName)
 	{
 		var fontWeight = FontWeight.Regular;
+		var fontStyle = TypefaceStyle.Normal;
+		var fontSlant = false;
+
+		var expectedTypeface = Typeface.CreateFromAsset(Application.Context.Assets, assetName);
+		Typeface expected;
+		if (OperatingSystem.IsAndroidVersionAtLeast(28))
+			expected = Typeface.Create(expectedTypeface, (int)fontWeight, fontSlant);
+		else
+			expected = Typeface.Create(expectedTypeface, fontStyle);
```

Comment:
Does this only work on < 28 because the default font weight is "Regular"?

Reply:
I think this is just an improved API. Pre 28 it was FontStyle (Normal, Bold, Italic, BoldItalic), now it is separate int weight and "is italic" properties.